### PR TITLE
Add capability for multiple enemies

### PR DESCRIPTION
**Pull request by Sindex42**

### CODE REVIEW

File: src/main.py [hunk 1]
```diff
@@ -44,21 +44,19 @@ def run(self):
...
-        self.all_sprites.add(self.hero)
-        self.enemy_sprites.add(self.enemy)
-
         for row, tiles in enumerate(self.map_data):
             for col, tile in enumerate(tiles):
                 if tile == '1':
                     self.walls_sprites.add(Wall(self, col, row))
+                if tile == 'E':
+                    self.enemy_sprites.add(Enemy(self, col, row))
+                if tile == 'H':
```
Comment (michaelnguyen974) [inline src/main.py:54]: Yummy refactoring

File: src/main.py [hunk 2]
```diff
@@ -44,21 +44,19 @@ def run(self):
...
-        self.walls_sprites = pg.sprite.Group()
-
-        self.all_sprites.add(self.hero)
-        self.enemy_sprites.add(self.enemy)
-
         for row, tiles in enumerate(self.map_data):
             for col, tile in enumerate(tiles):
                 if tile == '1':
                     self.walls_sprites.add(Wall(self, col, row))
+                if tile == 'E':
```
Comment (michaelnguyen974) [inline src/main.py:52]: Multiple enemies ;)

**Review by CKKH — APPROVED** [review]
Nice - lets look to extract map to its own class soon

**Review by hotblack86 — APPROVED** [review]
Ace refactoring!

File: src/map.txt [hunk 3]
```diff
@@ -1,12 +1,12 @@
 1111111111111111
+1E000000000000E1
 1000000000000001
+1000111001110001
+10001E0000E10001
 1000000000000001
-1011101110111001
-1010001010101001
-1011101110101001
+10000000H0000001
```
Comment (michaelnguyen974) [inline src/map.txt:7]: We shall make our map bigger ;)

**Review by michaelnguyen974 — APPROVED** [review]

File: src/map.txt (hunk 3 above)
Comment (matharotheelf) [reply src/map.txt:7]: Awesome to have enemies instantiated on map

**Review by matharotheelf — APPROVED** [review]
Awesome work

**Review by matharotheelf — APPROVED** [review]
cool

**Pull request merged**